### DISCUSSION TITLE
Where can I find a list of keyboard shortcuts for commafeed?

### DISCUSSION THREAD

**Discussion by holotone** [Athou/commafeed#1334] [Q&A] [answered]
Specifically I'm trying to find 'Mark all As Read' and 'Open Link', but a list would be great. I read a comment in an earlier discussion that ? had been implemented to show all keyboard shortcuts but that doesn't seem to work for me.

Thanks!

**Comment by Athou** [comment]
You can also find the list of keyboard shortcuts on the about page (https://www.commafeed.com/#/app/about).

**Reply by holotone** [reply]
Thanks a bunch!

**Reply by holotone** [reply]
Actually that URL just redirects to https://www.commafeed.com/#/welcome which does not have keyboard shortcuts.

**Reply by Athou** [reply, marked as answer]
Well you need to be logged in. If you're hosting CommaFeed yourself, you can visit the about page to get the list of shortcuts.

If you're not a CommaFeed user yet, you can use the demo account of commafeed.com by clicking "try the demo" or read the shortcuts in the source [here](https://github.com/Athou/commafeed/blob/master/commafeed-client/src/components/KeyboardShortcutsHelp.tsx)

**Reply by holotone** [reply]
Oohhh gotcha, you mean the about page on my install - Got it. Thanks again!
